modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/data/provider/test/DDMRESTDataProviderTest.java

```diff
@@ -583,6 +587,10 @@ private DDMDataProviderRequest _createDDMDataProviderRequest(
 			builder.withParameter("filterParameterValue", filterParameterValue);
 		}
 
+		if (locale != null) {
+			builder.withLocale(locale);
+		}
```

Comment:
@p-albuquerque `withLocale` should be before the `withParameter("filterParameterValue", filterParameterValue)`